### PR TITLE
Setting the icon, categories, and renaming debug configuration for VSCode extension.

### PR DESCRIPTION
**Pull request by dbalek**
Before going to [Marketplace](https://marketplace.visualstudio.com/), the Apache NetBeans Language Server extension should provide an icon and search categories. Renaming provided debug configuration from `Java (Polyglot)` to `Java 8+`.
Updating the records in package.json and corresponding ids in extension.ts.

### CODE REVIEW

**Review by JaroslavTulach — APPROVED** [review]
Extension with icon is nicer than without any icon. Thanks, Dušane!

**Review by JaroslavTulach — APPROVED** [review]

**Review by lkishalmi — CHANGES_REQUESTED** [review]
I'm just wondering how this one passed the license check of Travis...
Please update the licenseinfo.xml with the logo, just to be on the safe side.

**Comment by dbalek** [comment]
> I'm just wondering how this one passed the license check of Travis...
> Please update the licenseinfo.xml with the logo, just to be on the safe side.

Done.

File: java/java.lsp.server/vscode/package.json [hunk 1]
```diff
@@ -65,8 +65,8 @@
 		],
 		"debuggers": [
 			{
-				"type": "java-polyglot",
-				"label": "Java (Polyglot)",
+				"type": "java8+",
+				"label": "Java 8+",
```
Comment (JaroslavTulach) [inline java/java.lsp.server/vscode/package.json:69]: The "Java 8+" name seems to nicely stress the core functionality of Apache NetBeans Language Server VSCode extension - e.g. the ability to run on JDK8 (other extensions require JDK11). I think it is a good name.

**Review by JaroslavTulach — APPROVED** [review]

**Review by lkishalmi — APPROVED** [review]

**Pull request merged**